plugins/module_utils/ansiblemodule.py

```diff
@@ -3,4 +3,10 @@
 __metaclass__ = type
 
 
-from ansible.module_utils.basic import AnsibleModule  # noqa: F401
+try:
+    from ansible_collections.cloud.common.plugins.module_utils.turbo.module import (
+        AnsibleTurboModule as AnsibleModule,
+    )  # noqa: F401
+    AnsibleModule.collection_name = "kubernetes.core"
+except ImportError:
```

Comment:
LGTM, are we waiting for any additional changes.

Reply:
https://github.com/ansible-collections/cloud.common/pull/69 needs to be merged and released before the tests here will pass with turbo mode enabled. If the turbo mode tests are non-voting then we can merge and just have broken tests until the issue with cloud.common is resolved.